src/components/Patrons/Patrons.module.css

```diff
@@ -0,0 +1,59 @@
+.patrons {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  padding-top: 64px;
+  padding-bottom: 64px;
+  margin: 0;
+  background-color: var(--primary-background-color);
+}
+.patronsTitle {
+  color: var(--dark-text-color);
+  font-family: var(--decoraction-font);
+  align-self: flex-start;
+  font-size: 1.9375rem;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+  margin-bottom: 32px;
+  margin-left: 21px;
+}
+.patronsContainer {
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+}
+
+.patronsContainer > :last-child {
+  margin-top: 16px;
+}
+.patronsBox {
+  align-self: stretch;
+}
+.patronsImg {
+  width: 185px;
+  height: 131px;
```

Comment:
Height na obrazku to zły pomysł. Bez tego wysokość się nieznacznie zmienia, a na mniejszym vw obrazki nie są ucinane tylko się zmniejszają proporcjonalnie